no-computer.md

```diff
@@ -45,6 +53,7 @@ _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
+I dont think there are any other DSLs in this domain. This is because problem sets work so well!
```

Comment:
I think it is because problem sets cover such a large band of assigned work. Perhaps Sapling is one?